srcpkgs/lib3mf/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'lib3mf'
+pkgname=lib3mf
+version=2.2.0
+revision=1
+build_style=cmake
+configure_args="-DUSE_INCLUDED_LIBZIP=OFF -DUSE_INCLUDED_ZLIB=OFF
+ -DUSE_INCLUDED_GTEST=OFF -DUSE_INCLUDED_SSL=OFF
+ -DCMAKE_INSTALL_INCLUDEDIR='include/$pkgname' -DCMAKE_INSTALL_PREFIX=/usr
```

Comment:
> `-DCMAKE_INSTALL_INCLUDEDIR='include/$pkgname'`

this is the [default already](https://github.com/3MFConsortium/lib3mf/blob/v2.2.0/CMakeLists.txt#L23)

> `-DCMAKE_INSTALL_PREFIX=/usr`

this is [already specified](https://github.com/void-linux/void-packages/blob/master/common/build-style/cmake.sh#L54)

Reply:
Yeah, it was being a bit finicky so I tried adding those like the AUR build does.